src/components/Layout/Layout.scss

```diff
@@ -1,8 +1,23 @@
 .layout {
   display: grid;
-  grid-template-columns: 25% auto;
+  grid-template-columns: .25fr .25fr .25fr .25fr;
   max-width: 1440px;
   margin: 0 auto;
-  padding: 0 16px;
+  padding: 16px 16px;
   column-gap: 24px;
 }
+
+.layout__sidenav {
+  grid-column-start: 1;
+  grid-column-end: 1;
+}
+
+.layout__body {
+  grid-column-start: 2;
+  grid-column-end: 5;
+}
+
+.layout__full {
+  grid-column-start: 1;
+  grid-column-end: 5;
+}
```

Comment:
We need tablet and mobile logic - for sidenav and body - could you add some logic to stack the side nav and body when on tablet and mobile?